packages/dd-trace/src/appsec/iast/taint-tracking/rewriter.js

```diff
@@ -0,0 +1,66 @@
+'use strict'
+
+const Module = require('module')
+const shimmer = require('../../../../../datadog-shimmer')
+const log = require('../../../log')
+const TaintTrackingFilter = require('./filter')
```

Comment:
Why uppercased and not `taintTrackingFilter`? Is not a class that we are going to instanciate.